packages/react-avatar/src/components/Avatar/Avatar.tsx

```diff
@@ -1,24 +1,469 @@
 import * as React from 'react';
-import { useAvatar } from './useAvatar';
-import { AvatarProps } from './Avatar.types';
-import { makeClasses } from '@fluentui/react-utils';
-import { useInlineTokens } from '@fluentui/react-theme-provider/lib/compat/index';
+import { ax, makeStyles } from '@fluentui/react-make-styles';
 import { useFocusRects, nullRender } from '@fluentui/utilities';
-import * as classes from './Avatar.scss';
+
 import { Badge } from '../Badge/Badge';
+import { AvatarProps, AvatarState } from './Avatar.types';
 import { renderAvatar } from './renderAvatar';
+import { useAvatar } from './useAvatar';
+
+const animationDuration = {
+  duration50: '50ms',
+  duration100: '100ms',
+  duration150: '150ms',
+  duration200: '200ms',
+  duration300: '300ms',
+  duration400: '400ms',
+  duration500: '500ms',
+};
+
+const animationTiming = {
+  ultraFast: animationDuration.duration50,
+  faster: animationDuration.duration100,
+  fast: animationDuration.duration150,
+  normal: animationDuration.duration200,
+  slow: animationDuration.duration300,
+  slower: animationDuration.duration400,
+  ultraSlow: animationDuration.duration500,
+};
+
+const animationLines = {
+  decelerateMax: 'cubic-bezier(0.00,0.00,0.00,1.00)',
+  decelerateMid: 'cubic-bezier(0.10,0.90,0.20,1.00)',
+  decelerateMin: 'cubic-bezier(0.33,0.00,0.10,1.00)',
+  accelerateMax: 'cubic-bezier(1.00,0.00,1.00,1.00)',
+  accelerateMid: 'cubic-bezier(0.90,0.10,1.00,0.20)',
+  accelerateMin: 'cubic-bezier(0.80,0.00,0.78,1.00)',
+  maxEasyEase: 'cubic-bezier(0.80,0.00,0.20,1.00)',
+  easyEase: 'cubic-bezier(0.33,0.00,0.67,1.00)',
+  linear: 'linear',
+};
+
+const animations = {
+  fastOutSlowInMax: animationLines.decelerateMax,
+  fastOutSlowInMid: animationLines.decelerateMid,
+  fastOutSlowInMin: animationLines.decelerateMin,
+  slowOutFastInMax: animationLines.accelerateMax,
+  slowOutFastInMid: animationLines.accelerateMid,
+  slowOutFastInMin: animationLines.accelerateMin,
+  fastEase: animationLines.maxEasyEase,
+  normalEase: animationLines.easyEase,
+  nullEasing: animationLines.linear,
+};
+
+const useStyles = makeStyles<AvatarState>([
```

Comment:
We should put this in its own `useAvatarStyles` file to keep a separation of concerns.

Reply:
Yup, done 👍